<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/bkg.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="idlPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/main_bkg.jpg) 0 0 0 0 stretch stretch;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="PrizeLadderWidget" name="prizePage"/>
    <widget class="GameWindow" name="gamePage"/>
    <widget class="ResultWidget" name="resultPage"/>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResultWidget</class>
   <extends>QWidget</extends>
   <header>resultwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PrizeLadderWidget</class>
   <extends>QWidget</extends>
   <header>prizeladderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameWindow</class>
   <extends>QWidget</extends>
   <header>gamewindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
